src/main/java/address/controller/StatusBarHeaderController.java

```diff
@@ -13,26 +12,33 @@
     private StatusBar footerStatusBar;
 
     public StatusBarHeaderController() {
-        EventManager.getInstance().registerHandler(this);
         footerStatusBar = new StatusBar();
     }
 
     public StatusBar getFooterStatusBarView() {
         return footerStatusBar;
     }
 
-    @Subscribe
-    public void handleContactEditedEvent(ContactEditedEvent e) {
+    public void postStatus(ContactBaseStatus status) {
```

Comment:
Would it be better to rename the below methods (`postContactEditedStatus`, `postContactDeletedStatus` etc) to `postStatus`? Basically overloading `postStatus` instead of needing this method to multiplex it.